proto/tikvpb.proto

```diff
@@ -125,15 +127,29 @@ message BatchCommandsResponse {
             kvrpcpb.RawBatchPutResponse RawBatchPut = 16;
             kvrpcpb.RawDeleteResponse RawDelete = 17;
             kvrpcpb.RawBatchDeleteResponse RawBatchDelete = 18;
-            kvrpcpb.RawScanResponse RawScan = 19;    
+            kvrpcpb.RawScanResponse RawScan = 19;
             kvrpcpb.RawDeleteRangeResponse RawDeleteRange = 20;
             kvrpcpb.RawBatchScanResponse RawBatchScan = 21;
 
             coprocessor.Response Coprocessor = 22;
+
+            BatchCommandTestResponse Test = 255;
         }
     }
 }
 
 message BatchRaftMessage {
     repeated raft_serverpb.RaftMessage msgs = 1;
 }
+
+message BatchCommandTestRequest {
+    // Id of the test request
+    uint64 test_id = 1;
+    // The time between receiving the request and sending the response
+    uint64 delay_time = 2;
+}
+
+message BatchCommandTestResponse {
+    // Id of the test response, should be same with the one in the request
+    uint64 test_id = 1;
```

Comment:
The response is generated out of gRPC threads. So you can add
* `reply_time` which is for when the response is generated
* `reply_send_time` which is for when the response is sent out